src/components/dialog/dialog.stories.mdx

```diff
@@ -168,6 +168,52 @@ When mixing editable and non-editable content, you can use the <LinkTo kind='Des
   </Story>
 </Preview>
 
+### With help 
+
+<Preview>
+  <Story name="help" parameters={{ chromatic: { disable: true } }}>
```

Comment:
`with help` would match the naming on the fullscreen example, I also notice this is disabled for chromatic but the fullscreen example isn't I think either both should either be snapshotted or disabled